modules/nodes-descheduler-profiles.adoc

```diff
@@ -90,9 +90,70 @@ Do not enable `CompactAndScale` with any of the following profiles: `LifecycleAn
 
 endif::nodes[]
 ifdef::virt[]
-Use the `LongLifecycle` profile to enable the descheduler on a virtual machine. This is the only descheduler profile currently available for {VirtProductName}. To ensure proper scheduling, create VMs with CPU and memory requests for the expected load.
+Use the `DevKubeVirtRelieveAndMigrate` or `LongLifecycle` profile to enable the descheduler on a virtual machine.
+
+[IMPORTANT]
+====
+You can not have both `DevKubeVirtRelieveAndMigrate` and `LongLifeCycle` enabled at the same time.
+====
 endif::virt[]
 
+:FeatureName: The `DevKubeVirtRelieveAndMigrate` profile
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
```

Comment:
This should probably come within the _DevKubeVirtRelieveAndMigrate_ section. Maybe at [Line 111](https://github.com/openshift/openshift-docs/pull/92977/files#diff-6c9231033c58f628a3a3390a3ab8d8b44ffc7fd9eda66c424c718de5d151655bR111)?